.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - mvn test --batch-mode
+  - mvn checkstyle:check --batch-mode --fail-never
```

Comment:
What's `--fail-never` doing? Shouldn't it [failOnViolation](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#failOnViolation) so that the checks on GitHub will turn red when a PR regresses the Google Style adherence?